### PR TITLE
Updates to style guide guidelines

### PR DESCRIPTION
**Pull request by alexandrascript**
Adding guides for creating FAQs, release notes, and API guidelines.

I've also add links to talks given at various Write The Docs conferences when relevant.

### CODE REVIEW

File: docs/guide/writing/style-guides.md [hunk 1]
```diff
@@ -49,34 +51,104 @@ If you belong to an open-source community or NGO, you can consider the following
 
 Technical documentation takes many forms, each one targeting a certain need. Here, we can think of examples like tutorials, API documentation, and user manuals. So while many style guides adequately cover the main concerns of all technical documentation, sometimes a more specialized guide is needed.
 
-### Thinking about accessibility
+## Style for developers
+
+Developer documentation often have a specific set of rules in order to best meet their needs.
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md:54]: Personally, I'd use `has` here, and I find the `their` slightly confusing.

File: docs/guide/writing/style-guides.md [hunk 2]
```diff
@@ -49,34 +51,104 @@ If you belong to an open-source community or NGO, you can consider the following
...
 
-#### Code Snippets
+### API documentation
 
-- [Google code samples](https://developers.google.com/style/code-samples)
-- [Ruby / other languages](http://guides.rubyonrails.org/api_documentation_guidelines.html)
-- [Al language snippets](https://bocoup.com/blog/documenting-your-api)
+Clear, well-formated, and detailed API documentation is the key for developers to quickly consume and implement your API. It is also key to helping developers understand what happens when an API call is mad, and in the case of failure, understand what went wrong and how to fix it.
+
+The best API documentation is often the result of a designed API. Documentation cannot fix a badly planned API, and it is best to work on developing the API and the documentation concurrently.
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]:
```suggestion
The best API documentation is often the result of a well designed API. Documentation cannot fix a badly planned API, and it is best to work on developing the API and the documentation concurrently.
```

probably?

File: docs/guide/writing/style-guides.md [hunk 3]
```diff
@@ -49,34 +51,104 @@ If you belong to an open-source community or NGO, you can consider the following
...
+
+[Our guide to API documentation](https://www.writethedocs.org/guide/api/about-api-documentation/).
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]: Should we just fold this in? There doesn't feel enough in that link to call it a Guide. Or am I overthinking?

Comment (plaindocs) [reply docs/guide/writing/style-guides.md]: And by fold in I mean merge.

**Review by plaindocs — COMMENTED** [review]
This is great! Thanks @heyawhite

File: docs/guide/writing/style-guides.md (hunk 1 above)
Comment (alexandrascript) [reply docs/guide/writing/style-guides.md:54]: Absolutely. This has been addressed.

File: docs/guide/writing/style-guides.md (hunk 3 above)
Comment (alexandrascript) [reply docs/guide/writing/style-guides.md]: I agree. I'll go ahead and do that. Updates shortly.

Comment (alexandrascript) [reply docs/guide/writing/style-guides.md]: I've copied the relevant content and no longer link to that page. I've also removed it from the repo.

That said, there are still other files in the API directory that don't necessarily belong in the style guide.

File: docs/guide/writing/style-guides.md [hunk 4]
```diff
@@ -49,42 +49,134 @@ If you belong to an open-source community or NGO, you can consider the following
...
-A few great resources:
+### Accessibility resources
 
-- [MailChimp's style guide](https://styleguide.mailchimp.com/writing-for-accessibility/)
+- [MailChimp's writing style guide](https://styleguide.mailchimp.com/writing-for-accessibility/)
 - [A11Y Style Guide](https://a11y-style-guide.com/style-guide/)
+- [The Accessibility Cheatsheet](https://bitsofco.de/the-accessibility-cheatsheet/) by bitsofcode
 
-### More info is needed for the following
+Relevant talks from Write The Docs:
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]:
```suggestion
Relevant talks from Write the Docs:
```

File: docs/guide/writing/style-guides.md [hunk 5]
```diff
@@ -49,42 +49,134 @@ If you belong to an open-source community or NGO, you can consider the following
...
 
-#### User Guide / Manuals
+- [A11y-Friendly Documentation](https://www.writethedocs.org/videos/prague/2018/a11y-friendly-documentation-carolyn-stransky/) at Write The Docs Prague 2018
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]:
```suggestion
- [A11y-Friendly Documentation](https://www.writethedocs.org/videos/prague/2018/a11y-friendly-documentation-carolyn-stransky/) at Write the Docs Prague 2018
```

File: docs/guide/writing/style-guides.md [hunk 6]
```diff
@@ -49,42 +49,134 @@ If you belong to an open-source community or NGO, you can consider the following
...
+
+Some example release notes:
+
+- [Jira release notes](https://confluence.atlassian.com/jirasoftware/jira-software-release-notes-776821069.html)
+- [Slack](https://slack.com/release-notes/mac) and [more about their release note style](https://slackhq.com/a-little-thing-about-release-notes#)
+- [What's new in Google Ad Manager](https://support.google.com/admanager/answer/179039?hl=en)
+
+Related talks:
+
+- [Learning to love release notes](http://www.writethedocs.org/videos/prague/2018/learning-to-love-release-notes-anne-edwards/) at Write The Docs Prague 2018
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]:
```suggestion
- [Learning to love release notes](http://www.writethedocs.org/videos/prague/2018/learning-to-love-release-notes-anne-edwards/) at Write the Docs Prague 2018
```

File: docs/guide/writing/style-guides.md [hunk 7]
```diff
@@ -2,18 +2,18 @@
...
 
 Guides often help writers focus on the different readers of technical documentation, describing how to adapt content to different reader profiles, like developers, product managers, the general public, and others.
 
 Many focus on the language itself (tone, style, grammar).
 
-Others go beyond the content and discuss the organisation of the documentation, providing best practices on how to manage your content, version control, or publication and delivery strategies. While this is not what we focus on in our style guide, how your documentation is organized, and how easy it is for your readers to find what they are looking for, can be as important as the content itself.
+Others go beyond the content and discuss the organisation of the documentation, providing best practices on how to manage your content, version control, or publication and delivery strategies. While this is not what we focus on in this style guide for style guides, how your documentation is organized and the ease at which your readers can find what they are looking for, can be as important as the content itself.
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]: ease with which, maybe?

**Review by plaindocs — CHANGES_REQUESTED** [review]
Some super minor nits. :+1:

File: docs/guide/writing/style-guides.md (hunk 7 above)
Comment (alexandrascript) [reply docs/guide/writing/style-guides.md]: I think actually it should be "ease in which".

Comment (plaindocs) [reply docs/guide/writing/style-guides.md]: I'm not super convinced, but maybe it is a US/UK thing? Regardless not gonna die on this hill. Merging.

**Review by plaindocs — APPROVED** [review]

**Comment by plaindocs** [comment]
<3

**Comment by alexandrascript** [comment]
@plaindocs regarding the UK/US thing, that's something I noticed here in a few places (for example: organized/organised). I'm happy to standardize further contributions based on UK English, but there should probably be a consensus.

Thoughts?

**Comment by plaindocs** [comment]
Absolutely no reason for WTD to standardize on UK English. :-p 

I meant that if anything my feelings about 'with ease' / 'in ease' might have a UK English bent that I'm not aware of.

**Pull request merged**